lib/tests/access.rs

```diff
@@ -0,0 +1,608 @@
+mod parse;
+use parse::Parse;
+mod helpers;
+use helpers::new_ds;
+use regex::Regex;
+use surrealdb::dbs::Session;
+use surrealdb::iam::Role;
+use surrealdb::sql::Value;
+
+#[tokio::test]
+async fn access_bearer_database() -> () {
+	// TODO(gguillemas): Remove this once bearer access is no longer experimental.
+	std::env::set_var("SURREAL_EXPERIMENTAL_BEARER_ACCESS", "true");
+
+	let sql = "
+		-- Initial setup
+		DEFINE ACCESS api ON DATABASE TYPE BEARER;
```

Comment:
In any (or all) of these tests, should we include an `INFO FOR ...` statement, to validate the formatting of the defined access?

Reply:
What would you like to see there? The `INFO FOR` statement only shows information from access method itself without including grants. The display of the access method is already tested in `define.rs`, for example [here](https://github.com/surrealdb/surrealdb/blob/main/lib/tests/define.rs#L1686).

Reply:
Ah right! the `TYPE BEARER` formatting is currently not covered I think. Should we update the define tests in that case to include a test with `DEFINE ACCESS ... TYPE BEARER`?

Reply:
Fair enough :+1: